pkg/registry/views/projects-page.html

```diff
@@ -1,6 +1,84 @@
-<filter-bar class="content-filter">
-</filter-bar>
-
 <div>
-    <h1>Projects place holder</h1>
+    <table listing-table class="listing projects-listing">
+        <thead ng-if="!quiet">
+            <tr>
+                <td colspan="4">
+                    <span>
+                    <div id="add-button-icon">
```

Comment:
A <span> is a inline element. We shouldn't need to wrap <div> and <h3> in a span.